hw_3/task_for_homework_3.py

```diff
@@ -0,0 +1,56 @@
+text_sample = """homEwork:
+tHis iz your homeWork, copy these Text to variable.
+
+
+
+You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE 
+witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+
+
+it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE.
+
+
+
+last iz TO calculate nuMber OF Whitespace characteRS in this Tex. caREFULL, not only Spaces, 
+but ALL whitespaces. I got 87.
+"""
+# all text letters make lower-case
+text_lower = text_sample.lower()
+# split text by words
+c = text_lower.split()
+# check every word is it is equal to 'iz' and replace it to 'is'
+for i in c:
+    if i == 'iz':
+        index_of_iz = c.index(i)
+        c[index_of_iz] = 'is'
+# join all words into new string
+string_new = ' '.join(c)
+# split this new string into sentences
+string_newest = string_new.split('.')
+last_words = []
+# further split sentences into list of words and retrive last word from element in the list
+for i in string_newest:
+    if i:
+        words = i.split()
+        last_word = words[-1]
+        last_words.append(last_word)
```

Comment:
`[row.split()[-1] for row in string_newest if row]`

Reply:
Rewrote here https://github.com/OlenaEpam/data-repo/pull/7/commits/de9a2857cf7f4eb95e929b00962d37de3d3786d5